src/static/js/modules/footer-button.js

```diff
@@ -7,12 +7,12 @@
 var $ = require( 'jquery' );
 
 function init() {
-    var duration = 300;
+  var duration = 300;
 
-    $('.js-return-to-top').click( function( event ) {
-        event.preventDefault();
-        $('html, body').animate( { scrollTop: 0 }, duration );
-    } );
+  $( '.js-return-to-top' ).click( function( event ) {
```

Comment:
Should this use the `.on('click', ...);` method to stay consistent with other event handlers?